command/agent/event_endpoint.go

```diff
@@ -17,6 +17,103 @@ import (
 	"golang.org/x/sync/errgroup"
 )
 
+func (s *HTTPServer) EventSinksRequest(resp http.ResponseWriter, req *http.Request) (interface{}, error) {
+	if req.Method != http.MethodGet {
+		return nil, CodedError(405, ErrInvalidMethod)
```

Comment:
nit: there's https://golang.org/pkg/net/http/#pkg-constants we can use instead of raw numbers, e.g. `http.StatusMethodNotAllowed`